### PR TITLE
clone of the intuit.com sign up form

### PR DESCRIPTION
**Pull request by che30**
We worked with the new feature branch "secondbranch" and designed the inuit.com sign up "look-alike" form using HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    </div>
+                    <div class="inside">
+                        <label for="password">Password</label>
+                        <div class="lock">
+                            <input class="input-field ccc" type="password" id="password" placeholder=""> <span><i
+                                    class="fas fa-lock-open"></i></span>
+                        </div>
+                    </div>
+                    <button class="done" type="submit"><i class="fas fa-lock"></i> Create Account</button>
+                </form>
```
Comment (ghost) [inline index.html]: - The email should have an `<input>` of type `email`.

- The phone should have an `<input>` of type` tel`.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    background-color: #ECEEF1;
+}
+
+a {
+    text-decoration: none;
+    color: #0077C5;
+}
+
+main {
+    /* width: 100%; */
```
Comment (ghost) [inline style.css]: Remove comments in your css file.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+/* .insides i{
+    position: relative;
+    right: 5px;
+} */
+.input-field {
+    width: 97%;
+    height: 110%;
+    padding-top: 10px;
+    /* padding-bottom: 10px; */
+    /* background-color: ; */
```
Comment (ghost) [inline style.css]: Please remove these comments from your CSS file and any other comment that exists.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        </section>
+        <footer>
+            <div>
+                <h5>Legal Privacy Security <br> &#169; 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB,
+                    TurboTax, ProConnect and Mint
+                    are registered trademarks of Intuit Inc.<br>Terms and conditions, features, support, pricing, and
+                    service options subject to change without
+                    notice.</h5>
+            </div>
+        </footer>
```
Comment (ghost) [inline index.html:81]: The `<header>` and `<footer> `tag should not be inside the `<main>` tag.
```
<body>
    <header></header>
    <main></main>
    <footer></footer>
</body>
```

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Good job on your project. I enjoyed reviewing it.**

**There are a few things that need to be done before it gets approved.**

# Mandatory

1). **Your clone doesn't have the intuit logo above the form.**

 clone |  original
------- | ---------
intuit_clone | intuit_original

2). **It doesn't display the mint logo, quickbooks logo and turbotax logo as shown above.**

3). **Add a Getting started section in your readme file and provide details on how one can set up your repo on their local computer, and the necessary tools required. Check the [readme template](https://github.com/microverseinc/readme-template) for more details on this.**

### Please check for more comments below.

## Good luck 😃.

### PS: Incase you want to reach out, please send me a message on slack (@Rose Wanjohi)

File: style.css (hunk 2 above)
Comment (expjazz) [inline style.css]: - [ ] You need to remove all the comments.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                </div>
+                <div class="bottom">
+                    <h6>Invisible reCAPTCHA by Google <a href="#">Privacy Policy</a> and <a href="#">Terms of Use</a>.
+                    </h6>
+                </div>
+
+            </div>
+
+        </section>
+        <footer>
```
Comment (expjazz) [inline index.html:73]: - [ ] You need to do what was asked by the previous TSE. :pray: 

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+            <div class="logo"><img src="image/logo.png" alt="logo"></div>
+            <div class="center">
+                <div class="center-top">
+                    <h3>Create an Intuit account</h3>
+                    <h5>One account for everything Intuit, including Mint.<br><a href="#">Learn more</a></h5>
+
+                </div>
+                <form action="POST">
+                    <div class="insid"><label for="email">Email address</label>
+                        <input class="input-field" type="text" id="email"></div>
```
Comment (expjazz) [inline index.html]: - [ ] You need to change this to email. Just like it was asked before :pray:

File: index.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+        <header>
+            <div class="account">
+                <div class="acc">Already have an Inuit account? <br> <a class="acc-link" href="#">I forgot my User ID or
+                        Password</a></div>
+
+                <div class="ol"><i class="fas fa-lock"></i> Sign In</div>
+            </div>
+        </header>
+        <section>
+            <div class="logo"><img src="image/logo.png" alt="logo"></div>
```
Comment (expjazz) [inline index.html]: - [ ] This should be divided by three, with each part with the respective logo and text. 

image

File: README.md [hunk 8]
```diff
@@ -1 +1,50 @@
...
+
+👤 **Author2**
+
+- GitHub: [@che30](https://github.com/che30)
+- Twitter: [@BlanchardNsoh](https://twitter.com/BlanchardNsoh)
+- LinkedIn: [che nsoh](https://www.linkedin.com/in/che-nsoh-9455271b0/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```
Comment (expjazz) [inline README.md:56]: - [ ] You still need to add the section that was previously asked.

File: README.md [hunk 9]
```diff
@@ -1 +1,50 @@
...
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
```
Comment (expjazz) [inline README.md:69]: - [ ] You need to change this because it came from the template.

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hey team :+1: 

You did a great job here :100: 

The layout looks quite nice. However, it seems that you did not fixed what was previously requested in the last review. Check the inline comments below to see all the required changes with detail. Generally, you need to:

- [ ] remove all the comments

- [ ] Customize your readme

- [ ] Change your inputs

- [ ] Fix your HTML structure

Kindly check the inline comments for more details and changes to be made.

#### As stated above, the project looks really good.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: index.html [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+                    <h3>Create an Intuit account</h3>
+                    <h5>One account for everything Intuit, including Mint.<br><a href="#">Learn more</a></h5>
+
+                </div>
+                <form action="POST">
+                    <div class="insid"><label for="email">Email address</label>
+                        <input class="input-field" type="text" id="email"></div>
+                    <div class="insidee">
+                        <label for="phone">Phone (recommended)</label>
+                        <input class="input-field" type="number" id="phone">
```
Comment (umairarshadbutt) [inline index.html]: The phone should have an <input> of type tel.

File: index.html (hunk 7 above)
Comment (umairarshadbutt) [inline index.html]: - [ ] It shows the Mint logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

- [ ] It shows the Turbotax logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly implement the required changes as previous TSEs suggested.
- [ ] The phone should have an <input> of type tel.

- [ ] It shows the Mint logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

- [ ] It shows the Turbotax logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of ---- 💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**